wordle.py

```diff
@@ -1,6 +1,37 @@
 import random
+import sys
+from nltk.corpus import wordnet as wn
 from constants import GREEN, YELLOW, DEFAULT, CURSOR_UP, LINE_CLR
 
+CHOSEN_WORD = None
+HINT_MODE = False
+hints = []
+
+
+def remove_duplicates_and_truncate(*lists):
+    '''
+    Takes in a number of lists, removes duplicates from each of them,
+    truncates each of them to the first three elements, and returns a list
+    of all the resulting lists
+    '''
+    filtered_lists = []
+    for lis in lists:
+        lis = list(dict.fromkeys(lis))[:3]
+        filtered_lists.append(lis)
```

Comment:
read more on map/reduce in python and try if you can achieve the same using them

Reply:
Done. But we think not using map makes it more readable ?